services/wallet/transfer/commands.go

```diff
@@ -368,24 +367,11 @@ func (c *transfersCommand) Run(ctx context.Context) (err error) {
 	// Update MultiTransactionID from pending entry
 	for index := range allTransfers {
 		transfer := &allTransfers[index]
-		if transfer.MultiTransactionID == NoMultiTransactionID {
-			entry, err := c.transactionManager.GetPendingEntry(c.chainClient.ChainID, transfer.ID)
-			if err != nil {
-				if err == sql.ErrNoRows {
-					log.Warn("Pending transaction not found for", c.chainClient.ChainID, transfer.ID)
-				} else {
-					return err
-				}
-			} else {
-				transfer.MultiTransactionID = entry.MultiTransactionID
-				if transfer.Receipt != nil && transfer.Receipt.Status == types.ReceiptStatusSuccessful {
-					// TODO: Nim logic was deleting pending previously, should we notify UI about it?
-					err := c.transactionManager.DeletePending(c.chainClient.ChainID, transfer.ID)
-					if err != nil {
-						return err
-					}
-				}
-			}
+
+		entry, err := c.transactionManager.GetPendingEntry(c.chainClient.ChainID, transfer.ID)
+		if err == nil {
```

Comment:
And what if err is not nil ? Don't we need any log or return this err ?

Reply:
Great catch. Fixed it!